src/translations.js

```diff
@@ -33,11 +33,17 @@ module.exports = class Translation {
     } else {
       // if cache cannot be used , serve it from previous cache and update it in background.
       if (!_.isEmpty(self.translation[locale].value)) {
-        self.downloadTranslationData(locale).then((response) => {
-          self.updateCache(locale, response);
-        }).catch((e) => {
-          console.error(`Error while downloading translations ${locale}, ${e.stack}`);
-        });
+        let cembraLocales=["ch","it","fr","en"];
+        if(cembraLocales.indexOf(locale)>-1){
+          this.getTranslationsFromLokalise();
```

Comment:
i think what i remember you need to update the cache also right ?

Reply:
yes

Reply:
I am updating the cache in that method